include/HUDL/HUDL.hpp

```diff
@@ -77,6 +76,14 @@ class HUDL {
         BOTTOM_RIGHT
     };
 
+    /**
+     * The node ID used to identify the device on the CAN network.
+     */
+    static constexpr uint8_t NODE_ID = 0x11;
```

Comment:
All the node IDs should be in decimal, not hex because the integration team uses decimal. For the HUDL, it didn't matter before because it didn't send a heartbeat, but we should fix this now
```suggestion
    static constexpr uint8_t NODE_ID = 11;
```